android/runtime/common/src/js/console.js

```diff
@@ -44,3 +45,27 @@ exports.error = function () {
 exports.debug = function () {
 	Titanium.API.debug(join(arguments));
 };
+
+exports.time = function (label) {
+	if (times[label]) {
+		exports.warn('Label ' + label + ' already exists');
```

Comment:
imo, this would read nicer using template literals. Which are supported by the v8 version I believe. What're our best practices for the JS code on Android. The eslint options specify ES5

```json
  "parserOptions": {
    "ecmaVersion": 5,
    "sourceType": "script"
  }
```

Reply:
I'm fine with that since it's supported.